mod_test.ts

```diff
@@ -935,10 +935,10 @@ Deno.test("custom plugin for scheme with import map", async (t) => {
   });
 });
 
-const TXT_PLUGIN: esbuildNative.Plugin = {
+const CSS_PLUGIN: esbuildNative.Plugin = {
```

Comment:
I changed this to be a CSS plugin instead of a TXT plugin since esbuild has a default loader for txt files that behaves the same way. So the txt plugin could have given a false positive if it was implemented incorrectly since it would end up using the default loader for txt.